aip/0217.md

```diff
@@ -66,10 +72,17 @@ request with an error.
 When paginating over a list, it is likely that the service will not know that
 there are unreachable locations or resources initially.
 
-- The response **may** provide the unreachable location or resource in the
-  `unreachable` field on the page where it is initially encountered.
 - The response **must** provide any outstanding unreachable locations or
-  resources in the `unreachable` field on the final page.
+  resources in the `unreachable` field on the _final_ page.
+  - The response **should not** include both requested data and unreachable
+    resources on the same page.
+    - For example, if there are two pages of books and one unavailable
+      publisher, there should be three pages total: first the two pages of
+      books, and then a final page with no books and the unavailable publisher.
+  - If the number of unavailable resources to list is very large, the response
```

Comment:
With the above suggested change, this would just be "Responses containing unreachable resources should honor the page_size field in the same way as for the resources".

(Note the change from unavailable to unreachable as well.)